src/Uno.UI/UI/Xaml/Shapes/Shape.Skia.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Collections.Specialized;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Uno.Extensions;
+using Windows.UI.Composition;
+using Uno.Disposables;
+using System.IO.Compression;
+
+namespace Windows.UI.Xaml.Shapes
+{
+	[Markup.ContentProperty(Name = "SvgChildren")]
+	partial class Shape
+	{
+		private ShapeVisual _rectangleVisual;
+		private SerialDisposable _fillSubscription = new SerialDisposable();
+		private SerialDisposable _strokeSubscription = new SerialDisposable();
+		private CompositionSpriteShape _pathSpriteShape;
+
+		public Shape()
+		{
+			_rectangleVisual = Visual.Compositor.CreateShapeVisual();
+			Visual.Children.InsertAtBottom(_rectangleVisual);
+
+			InitCommonShapeProperties();
+		}
+
+		internal virtual SkiaGeometrySource2D GetGeometry(Size finalSize)
+		{
+			throw new NotSupportedException($"Path.GetGeometry must be implemented");
+		}
+
+		private void InitCommonShapeProperties() { }
+
+		protected override Size MeasureOverride(Size availableSize)
+		{
+			var geometrySource = GetGeometry(availableSize);
+
+			return new Size(geometrySource.Geometry.Bounds.Width, geometrySource.Geometry.Bounds.Height);
+		}
+
+		protected override Size ArrangeOverride(Size finalSize)
+		{
+			var area = new Rect(0, 0, finalSize.Width, finalSize.Height);
+
+			switch (Stretch)
+			{
+				default:
```

Comment:
Codacy found an issue: [Remove this empty 'default' clause.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)